### PR TITLE
Fix expect tests on windows

### PR DESCRIPTION
**Pull request by hhugo**
Attempt to fix the windows CI job

### CODE REVIEW

**Comment by hhugo** [comment]
@toots, do you know why other CI jobs don't show up ?

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -38,6 +41,20 @@ jobs:
...
       - name: Use OCaml ${{ matrix.ocaml-compiler }}
+        if: runner.os == 'Windows'
+        uses: ocaml/setup-ocaml@v2
+        with:
+          ocaml-compiler: ${{ matrix.ocaml-compiler }}
+          opam-repositories: |
+            dra27: https://github.com/dra27/opam-repository.git#windows-5.0
+            default: https://github.com/ocaml-opam/opam-repository-mingw.git#sunset
+            opam: https://github.com/ocaml/opam-repository.git
+          dune-cache: ${{ matrix.os == 'ubuntu-latest' }}
```
Comment (toots) [inline .github/workflows/build.yml:52]:
```suggestion
          dune-cache: false
```

**Comment by hhugo** [comment]
@toots, I've extended the scope of the PR to "cleanup" the CI. Do you have any objection remove ci.yml ?

File: .github/workflows/build.yml (hunk 1 above)
Comment (hhugo) [reply .github/workflows/build.yml:52]: Why that ?

**Comment by hhugo** [comment]
I've tested the fix in #147

**Comment by toots** [comment]
This has been merged with the other one.

**Pull request closed without merging**